spharpy/classes/audio.py

```diff
@@ -241,6 +239,22 @@ def __init__(self, data, times, basis_type, normalization,
         TimeData.__init__(self, data=data, times=times, comment=comment,
                           is_complex=is_complex)
 
+    @classmethod
+    def from_spherical_harmonics_definition(
+            cls, data, times, sh_definition, domain='time',
+            fft_norm='none', comment="", is_complex=False):
```

Comment:
```suggestion
    def from_definition(
            cls, sh_definition,  data, times, domain='time',
            fft_norm='none', comment="", is_complex=False):
```
I'd suggest to change the order and shorten the name as in #249